src/app/win32_service.cc

```diff
@@ -152,7 +148,8 @@ void I2PService::Start(
 void I2PService::OnStart(
     DWORD argc,
     PSTR *argv) {
-  // TODO(unassigned): unused args
+  if (argc || argv)
+    LOG(error) << "I2PService: Unknown startup arguments.";
```

Comment:
Another assumption that the Windows service did not use startup arguments. Instead of silently ignoring the non-use, I decided to log the error, hoping this would provide a hint to implement startup argument use.

I'll investigate the proper use, or non-use, of Windows service startup arguments.

Reply:
Thank you.